docs/api.rst

```diff
@@ -85,15 +85,14 @@ Accepts an object representing options with the values to updated. Other options
     tabReplace: '    ', // 4 spaces
     classPrefix: ''     // don't append class prefix
                         // … other options aren't changed
-  })
+  });
   hljs.initHighlighting();
 
 
 ``initHighlighting()``
 ----------------------
 
-Applies highlighting to all ``<pre><code>..</code></pre>`` blocks on a page.
-
+Applies highlighting to all ``<pre><code>…</code></pre>`` blocks on a page.
```

Comment:
I don't know about this change...

Reply:
Forgive me about my perfections. Do I need to revert it back?

Reply:
Maybe I'm just old school for preferring ASCII... I'd have turned it into three periods personally.  If you don't mind, but I don't care enough to force the issue.  Is it common to use the ... thingy in mostly ASCII text documents?

Reply:
> Is it common to use the ... thingy in mostly ASCII text documents?

I don&rsquo;t think so. But some users prefer post-parsing those characters into some typographical characters like `…`, `‘`, `’`, `“`, `”`, `–` and `—` in the HTML output.

Will convert it to `...` ASAP!

Reply:
>  But some users prefer post-parsing those characters into some typographical characters like

Yeah I've always thought of it as a post-parsing concern of the renderer...